### PR TITLE
Feat/beast improvements

### PR DESCRIPTION
**Pull request by spaghettiOnToast**
UI styles, fixes and feats.

Public
- Added Images

Components
- Actions: clean up
- BattleScene: removed
- Beast: styles and images
- BeastDisplay: styles and icons
- DiscoveryDisplay: styles and icons
- GameData: images
- Info: styles
- ItemDisplay: add +1 luck to jewellery items and styles
- Leaderboard: styles
- LootIcon: add size
- Marketplace: add claim all button
- MarketplaceRow: clean up
- page: clean up and disable buttons when in battle

### CODE REVIEW

File: ui/package-lock.json
Comment (ponderingdemocritus) [inline ui/package-lock.json:1]: please use yarn not npm to install. Having the two lock files sometimes breaks deployment.

File: ui/public/beasts/ghoul.png
Comment (ponderingdemocritus) [inline ui/public/beasts/ghoul.png:1]: should this be here?

Comment (spaghettiOnToast) [reply ui/public/beasts/ghoul.png:1]: This was a placeholder image as we don't have one for this beast yet.

**Comment by spaghettiOnToast** [comment]
- Added more ui/styles fixes
- Gold balance fix on market
- Equip button disabling

**Pull request merged**